src/main/java/org/broadinstitute/hellbender/engine/spark/GATKRegistrator.java

```diff
@@ -30,6 +33,10 @@ public void registerClasses(Kryo kryo) {
 
         // JsonSerializer is needed for the Google Genomics classes like Read and Reference.
         kryo.register(Read.class, new JsonSerializer<Read>());
+
+        kryo.register(Arrays.asList("some","values").getClass(), new ArraysAsListSerializer());
+        kryo.register(Collections.nCopies(2,"").getClass(), new JavaSerializer());
```

Comment:
done, I've actually removed the ArraysAsList serializer because it turns out it's not doing what I was hoping.